tests/bdd/scale/test_NAS_T1123.py

```diff
@@ -0,0 +1,142 @@
+    driver.find_element_by_xpath('//button[@ix-auto="button__SAVE"]').click()
+
+
+@then(parsers.parse('"{smbname}" should be added, Click on service and the Service page should open'))
+def mysmbshare_should_be_added_click_on_service_and_the_service_page_should_open(driver, smbname):
+    """"{smbname}" should be added, Click on service and the Service page should open."""
+    #assert wait_on_element(driver, 5, '//div[contains(.,"SMB")]')
+    assert wait_on_element(driver, 5, f'//div[contains(.,"mysmbshare")]')
+    time.sleep(2)
+    #assert wait_on_element(driver, 7, '//button[@ix-auto="button__ENABLE SERVICE"]', 'clickable')
+    #driver.find_element_by_xpath('//button[@ix-auto="button__ENABLE SERVICE"]').click()
+    #time.sleep(2)
+    #assert wait_on_element(driver, 7, '//div[contains(text(),"The SMB service has been enabled.")]')
+    #driver.find_element_by_xpath('//span[contains(text(),"Close")]').click()
+
+
+
+@then('If the SMB serivce is not started start the service, and click on SMB Start Automatically checkbox')
+def if_the_smb_serivce_is_not_started_start_the_service_and_click_on_smb_start_automatically_checkbox(driver):
+    """If the SMB serivce is not started start the service, and click on SMB Start Automatically checkbox."""
+    time.sleep(1)
+    #assert wait_on_element(driver, 5, '//services')
+    # Scroll to SMB service
+    #element = driver.find_element_by_xpath('//div[contains(text(),"WebDAV")]')
+    #driver.execute_script("arguments[0].scrollIntoView();", element)
+    #time.sleep(1)
+    #driver.find_element_by_xpath('//div[contains(text(),"SMB")]')
+    #value_exist = attribute_value_exist(driver, '//mat-slide-toggle[@id="slide-toggle__state_SMB"]', 'class', 'mat-checked')
+    #if not value_exist:
+    #    driver.find_element_by_xpath('//mat-checkbox[@ix-auto="overlay__SMB_Running"]').click()
+    #time.sleep(2)
+
+
+@then(parsers.parse('Send a file to the share with nas_ip/"{mysmbshare}" and "{user}" and "{password}"'))
+def send_a_file_to_the_share_with_nas_ipmysmbshare_and_administrator_and_abcd1234(driver, nas_ip, mysmbshare, user, password):
+    """Send a file to the share with nas_IP/"{mysmbshare}" and "{user}" and "{password}"."""
+    run_cmd('touch testfile.txt')
+    time.sleep(2)
+    results = run_cmd(f'smbclient //{nas_ip}/{mysmbshare} -W AD01 -U {user}%{password} -c "put testfile.txt testfile.txt"')
+    time.sleep(1)
+    assert results['result'], results['output']
+
+
+@then('Verify that the is on nas_ip with root and password')
+def verify_that_the_is_on_nas_ip_with_root_and_password(driver, nas_ip, root_password):
+    global results
+    cmd = 'ls -la /mnt/system/my_acl_dataset/'
+    results = ssh_cmd(cmd, 'root', root_password, nas_ip)
+    assert results['result'], results['output']
+    assert 'testfile' in results['output'], results['output']
+
+    ## return to dashboard
+    assert wait_on_element(driver, 10, '//mat-list-item[@ix-auto="option__Dashboard"]', 'clickable')
```

Comment:
Since the next test will go back to the dashboard You should not go to the dashboard.

Try to avoid loading the UI as much as possible.